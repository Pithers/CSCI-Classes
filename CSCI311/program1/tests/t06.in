table table-01 1 Julia
table table-02 1 Julia
table table-03 1 Julia
table table-04 1 Julia
table table-05 1 Julia
table table-06 1 Julia
table table-07 1 Julia
table table-08 1 Julia
table table-09 1 Julia
table table-10 1 Julia
table table-11 1 Julia
table table-21 1 Julia
table table-31 1 Julia
table table-41 1 Julia
table table-51 1 Julia
table table-61 1 Julia
table table-71 1 Julia
table table-81 1 Julia
table table-91 1 Julia
table table-12 1 Julia
table table-13 1 Julia
table table-14 1 Julia
table table-15 1 Julia
table table-16 1 Julia
table table-17 1 Julia
table table-18 1 Julia
table table-19 1 Julia
table table-22 1 Julia
table table-23 1 Julia
table table-24 1 Julia
table table-25 1 Julia
table table-26 1 Julia
table table-27 1 Julia
table table-28 1 Julia
table table-29 1 Julia
table table-32 1 Julia
table table-33 1 Julia
table table-34 1 Julia
table table-35 1 Julia
table table-36 1 Julia
table table-37 1 Julia
table table-38 1 Julia
table table-39 1 Julia
table table-42 1 Julia
table table-43 1 Julia
table table-44 1 Julia
table table-45 1 Julia
table table-46 1 Julia
table table-50 2 Carl
party 2 Bobby 100
end
