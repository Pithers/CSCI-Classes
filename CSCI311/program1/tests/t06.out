Bobby seated at table-50
Bobby finished at table-50
Carl served 2
